src/format_meta.rs

```diff
@@ -50,9 +50,7 @@ impl<'a> Meta<'a> {
             let scaled_max = minmaxscale(range.max as usize, 0, buffer.len());
             let scaled_size = minmaxscale(size as usize, 0, buffer.len());
             let mut chars = SCALE;
-            for i in scaled_min..scaled_max {
-                chars[i] = '-';
-            }
+            chars[scaled_min..scaled_max].fill('-');
```

Comment:
very nice

Reply:
And it should result in better generated code. Yes, yes, it's completely pointless optimizations, but it reads clearer to me.